services/verification.rst

```diff
@@ -56,7 +56,8 @@ GPFS is available on the login node ``lsst-dev01`` and on all of the compute nod
 :file:`/scratch/<username>` 
 
 ready and available for use.  The per user :file:`/scratch` space is volatile with a 180 day purge policy
-and is not backed up. 
+and is not backed up.
+In addition, the :file:`/project` space is mounted; it is volatile, not backed up, and has no purge policy; cooperative self-management of this space is expected.
```

Comment:
Do we get `/project/<username>`?

Mention `/project` in the paragraph above that lists `For convenience...`